packages/@tinacms/fields/src/plugins/GroupListFieldPlugin.tsx

```diff
@@ -62,7 +63,7 @@ interface GroupFieldDefinititon extends Field {
      */
     label?: string
   }
-  allowItemDuplication?: boolean
+  onItemDuplication?: (name: string, index: number, value: any, callback: Mutators['insert']) => void
```

Comment:
Great use of types

Reply:
Actually, can we rename that from callback to `formInsert` or something more relevant? callback is too generic